### ISSUE TITLE
Change the MNIST network archecture to a CNN

### ISSUE THREAD

**Issue by johncoopertr** [johncoopertr/FraudNet#24]
Alter the MNIST network architecture to a CNN that looks like this

The Architecture Breakdown:

<img width="1089" height="1246" alt="Image" src="https://github.com/user-attachments/assets/3d6760f8-d9df-4b0c-90db-9337a3371107" />

Something like the following:

        Layer (type)               Output Shape         Param #
================================================================
         ZeroPad2d-1            [-1, 1, 32, 32]               0
            Conv2d-2           [-1, 16, 28, 28]             416
       BatchNorm2d-3           [-1, 16, 28, 28]              32
              ReLU-4           [-1, 16, 28, 28]               0
         MaxPool2d-5           [-1, 16, 14, 14]               0
           Flatten-6                 [-1, 3136]               0
            Linear-7                   [-1, 10]          31,370
           Softmax-8                   [-1, 10]               0
================================================================
Total params: 31,818
Trainable params: 31,818
Non-trainable params: 0

Remember to use the `mnist-train.parquet` and `mnist-test.parquet` files in the root for training and testing, respectively.
